index.html

```diff
@@ -225,7 +225,7 @@ <h2>Contact Me</h2>
             </p>
         </div>
         <div class="form">
-            <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgdqjvg" method="post">
+            <form class="contact-form" id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgdqjvg" method="post">
                 <fieldset id="fs-frm-inputs">
                     <label class="label" for="full-name">Full Name</label>
                     <input type="text" name="name" id="full-name" required maxlength="30" placeholder="Oshie" />
```

Comment:
- [ ] You should implement the following interactions:
  - When the user changes the content of any input field, the data is saved to the local storage.
  - When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.
Your project is still missing the above [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md), please kindly implement them.